readthedocs/builds/managers.py

```diff
@@ -179,3 +179,38 @@ class InternalBuildManager(SettingsOverrideObject):
 
 class ExternalBuildManager(SettingsOverrideObject):
     _default_class = ExternalBuildManagerBase
+
+
+class VersionAutomationRuleManager(PolymorphicManager):
+
+    def append_rule(
```

Comment:
For consistency: https://github.com/readthedocs/readthedocs.org/pull/5998#discussion_r309640219